cmd/vicadmin/server.go

```diff
@@ -168,36 +357,35 @@ func (s *server) stop() error {
 	return nil
 }
 
+func getSessionCookie(req *http.Request) *sessions.Session {
+	cookieSessionData, _ := store.Get(req, sessionCookieKey)
+	return cookieSessionData
+}
+
 func (s *server) bundleContainerLogs(res http.ResponseWriter, req *http.Request, f format) {
 	defer trace.End(trace.Begin(""))
 
 	readers := defaultReaders
+	c, err := s.getSessionFromRequest(req)
```

Comment:
I will be adding error handling to check and make sure `err` is non-nil here. This is still prototype version.